bin/ulauncher-toggle

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/sh
+#!/usr/bin/env sh
```

Comment:
Unfortunately this breaks the build for Fedora. I've already tried this.
I'll think what we can do here